manila/message/message_field.py

```diff
@@ -144,6 +144,12 @@ class Detail(object):
           "ticket."))
     FILTER_AFFINITY = ('027', FILTER_MSG % 'AffinityFilter')
     FILTER_ANTI_AFFINITY = ('028', FILTER_MSG % 'AntiAffinityFilter')
+    SHARE_NETWORK_PORT_QUOTA_LIMIT_EXCEEDED = (
+        '029',
+        _("Share Driver failed to create share due share network port "
+          "quota limit exceeded. You may increase quota of share network "
+          "port and retry your request. If this doesn't work, contact your "
+          "administrator to troubleshoot issues with your network."))
```

Comment:
```suggestion
        _("Share Driver failed to create share server on share network "
          "due to port limit exceeded. You may increase quota of network "
          "ports and retry your request. If this doesn't work, contact your "
          "administrator to troubleshoot issues with your network."))
```

Reply:
updated. Does this needs to upstream ?

Reply:
Thanks, yes - this is an improvement we can give back to the community. Nothing special to SAP at this :)